src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -304,14 +304,14 @@ public async Task<RestoreResult> ExecuteAsync(CancellationToken token)
                     });
                 }
 
-                AuditUtility.EnabledValue enableAudit = AuditUtility.ParseEnableValue(
+                bool auditEnabled = AuditUtility.ParseEnableValue(
                     _request.Project.RestoreMetadata?.RestoreAuditProperties?.EnableAudit,
                     _request.Project.FilePath,
                     _logger);
-                telemetry.TelemetryEvent[AuditEnabled] = AuditUtility.GetString(enableAudit);
-                if (enableAudit != AuditUtility.EnabledValue.ExplicitOptOut)
+                telemetry.TelemetryEvent[AuditEnabled] = auditEnabled ? "enabled" : "disabled";
```

Comment:
I discussed this telemetry change with @jebriede, as we don't believe that knowing "implicitly enabled" vs "explicitly enabled" is sufficiently useful.  @jebriede also gave me feedback that he thinks introducing this breaking change in the telemetry will be better off in the long term, rather than reporting "ExplicitOptIn" and "ExplicitOptOut" as the only two values we end up reporting.

If explicit vs implicit opt in is useful, we can add it, by setting a second MSBuild property (with underscore property name). In order to implement this, we'd need to change the VS Nomination code (and therefore also create a pull request on dotnet/project-system), and `LegacyPackageReferenceProject`. I'm not entirely sure if we'd need to also change the DGSpec reader/writer.

We only report telemetry in VS, so the additional MSBuild property wouldn't be needed for CLI scenarios, however, we don't have any other VS specific MSBuild properties. And if the VS scenario does require changes to the DGSpec, then the CLI restore, VS no-op restore scenerio will break. So, if we'd want to keep everything consistent, we'd also need to change static and non-static graph restore. I think the DGSpec changes will make nuget.exe restore current, but there's a chance that nuget.exe might need a change too.

So, as we can see, there's a high cost to keeping this information. Therefore, we need a good justification for why it would be useful if we want to keep it.